cmd/crio/config.go

```diff
@@ -61,10 +65,7 @@ it later with **--config**. Global options will modify the output.`,
 		}
 
 		if c.IsSet("migrate-defaults") {
-			logrus.Infof("Migrating config from %s", from)
-			if err := migrate.Config(conf, from); err != nil {
```

Comment:
can we remove the dead code in the migrate package as well please?

Reply:
updated!

```
❯ ./bin/crio config --default -m 1.17
INFO[2023-05-09 09:01:06.062276787-05:00] Starting CRI-O, version: 1.27.0, git: 0b497615debb4c99dea13d251ba364074099c95e(dirty)
FATA config migration has been deprecated and will be removed in v1.29.0

❯ ./bin/crio config --default
INFO[2023-05-09 09:01:09.023879670-05:00] Starting CRI-O, version: 1.27.0, git: 0b497615debb4c99dea13d251ba364074099c95e(dirty)
INFO Using default capabilities: CAP_CHOWN, CAP_DAC_OVERRIDE, CAP_FSETID, CAP_FOWNER, CAP_SETGID, CAP_SETUID, CAP_SETPCAP, CAP_NET_BIND_SERVICE, CAP_KILL
# The CRI-O configuration file specifies all of the available configuration
# options and command-line flags for the crio(8) OCI Kubernetes Container Runtime
# daemon, but in a TOML format that can be more easily modified and versioned.
#
...
```